### PR TITLE
Feature/view habit events

### LINKED ISSUE #131: Delete habit events in db when deleting habit after new updates
After swipe functionality updates, habit events no longer were deleted when a habit was deleted. Let's fix this so we don't cause db clutter.

### PR DESCRIPTION
**Pull request by JuliePilz**
### What is it?
This PR Adds functionality to view habit events by clicking on them in the habit events list. It also updates the swipe functionality, as well as re-fixing the db bug related to deleting habits. 

closes #131 , #42 and #116

### CODE REVIEW

File: app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitListFragment.java
Comment (jaskim9824) [inline app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitListFragment.java:284]: Minor: If this method is going to be used only within this class, then it should be private.

**Review by jaskim9824 — APPROVED** [review]
Left a comment about a minor concern, but overall looks great!

File: app/src/androidTest/java/com/example/habitsmasher/MainActivityTest.java
Comment (rudy-patel) [inline app/src/androidTest/java/com/example/habitsmasher/MainActivityTest.java:1280]: This should probably be removed
- [ ] remove commented lines

File: app/src/androidTest/java/com/example/habitsmasher/MainActivityTest.java
Comment (rudy-patel) [inline app/src/androidTest/java/com/example/habitsmasher/MainActivityTest.java:1283]: I think @jaskim9824's PR adding the wait already went in, so we probably don't need this anymore as this would be 8 total seconds of just idling in **all of our tests**.

File: app/src/main/res/layout/fragment_habit_event_view.xml
Comment (rudy-patel) [inline app/src/main/res/layout/fragment_habit_event_view.xml:82]: I'm assuming the reason behind why this isn't extracted into a string resource is because it's temporary, in which case this is fine

**Review by rudy-patel — CHANGES_REQUESTED** [review]
Looks good, we should probably take out the lines that you used for personal testing in the UI test, along with removing the **sleep** line in the UI test (as Jason's PR is already merged in). Probably a good idea to merge in develop one last time to verify all UI tests pass, instead of having each test run **8 seconds longer** than we need it to.

After that, it looks fine to me, so I'll approve!

**Review by Nova-Drive — COMMENTED** [review]
Looks good, but all of my concerns I share with Rudy. Once those are addressed you have my approval.

File: app/src/main/res/layout/fragment_habit_event_view.xml
Comment (JuliePilz) [reply app/src/main/res/layout/fragment_habit_event_view.xml:82]: Correcto

File: app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitListFragment.java
Comment (JuliePilz) [reply app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitListFragment.java:284]: Good catch!

File: app/src/androidTest/java/com/example/habitsmasher/MainActivityTest.java
Comment (rudy-patel) [reply app/src/androidTest/java/com/example/habitsmasher/MainActivityTest.java:1280]: Any updates on this?

Comment (JuliePilz) [reply app/src/androidTest/java/com/example/habitsmasher/MainActivityTest.java]: Just running UI tests

**Review by jaskim9824 — APPROVED** [review]

**Review by Nova-Drive — APPROVED** [review]
Nice work!

File: app/src/androidTest/java/com/example/habitsmasher/MainActivityTest.java
Comment (rudy-patel) [reply app/src/androidTest/java/com/example/habitsmasher/MainActivityTest.java]: Can we remove these lines before we merge this into `develop`?

**Review by rudy-patel — CHANGES_REQUESTED** [review]
The commented-out lines for your test user are still in the UI test class, let's make sure to get rid of that before we merge this in. Other than that, it looks good to me.

**Review by rudy-patel — APPROVED** [review]

**Pull request merged**
